src/context/authContext.js

```diff
@@ -31,8 +32,10 @@ const AuthProvider = ({
                 setUser(response.data.foundUser);
                 navigate("/homepage")
             }
+            toast.success("Login Successful")
         } catch (err) {
             console.log(err)
```

Comment:
is there need for logs , if we have toast?

Reply:
Toast is for the user, console is for me to get the actual error. I'll remove it though